vmdb/app/views/ops/_rbac_tag_box.html.haml

```diff
@@ -3,7 +3,7 @@
   %table.style1
     %tr
       %td.key
-        = session[:customer_name]
+        = current_brand.customer_name
```

Comment:
@Fryguy : those methods in application_helper, it's just mess, it does not scale. It would make a bit of sense if there was some BrandHelper (that would be the "Rails Way", but I don't like it either). 

But in ApplicationHelper, it would be just another piece of junk at the level of globals as @kbrock wrote.

If you really want to move the presentatin logic outside the model, create a simple class:
```
class BrandPresenter
   def initialize(brand)
      @brand = brand
   end
   def customer_name
     ...
    end
  end
end
```

Then call the view with locals: ``render .... :localst => { current_brand => BrandPresenter.new(brand) }``

And in the view do:
```
      = current_brand.customer_name
```

Than way it's all nicely separated and explicit.